src/resources/s3.js

```diff
@@ -77,6 +77,29 @@ class S3Bucket { // eslint-disable-line padded-blocks
 		};
 	}
 
+	/**
+	 * Translate the logging configuration into the 'logging params' for the AWS SDK.
+	 *
+	 * @param {string} bucketName the bucket name
+	 * @param {PublicAccessBlockConfiguration} publicAccessBlockConfiguration Public Access Block configuration
```

Comment:
Please document the type (see for example `LoggingConfiguration`)

Reply:
Fixed by https://github.com/Collaborne/kubernetes-aws-resource-service/pull/43/commits/2e686d454a0b5b5bee247c946f156700a869eb8c